spec/fixtures/application/1.0/application.json

```diff
@@ -111,12 +136,21 @@
           "payment_type": "other",
           "amount": 2500,
           "frequency": "annual",
+          "ownership_type": "applicant",
           "metadata": { "details": "Book royalty" }
         },
         {
           "payment_type": "board_from_family",
           "amount": 1603,
           "frequency": "week",
+          "ownership_type": "applicant",
```

Comment:
These income screens are join partner and applicant screens

Reply:
It's separate partner and applicant screens, it's only joint for outgoings payments

Reply:
Yep, not paying attention, you're right. I was reading it as if this was the pays rent/mortgage pages sorry!

Reply:
No worries!